### PR TITLE
Added Rolls to brands index

### PR DESCRIPTION
**Pull request by KennethYoel**

### CODE REVIEW

File: data/brands/amenity/restaurant.json [hunk 1]
```diff
@@ -2613,6 +2613,19 @@
...
     },
+    {
+      "displayName": "Rolls",
+      "id": "rolls-fe858e",
+      "locationSet": {"include": ["fi"]},
+      "tags": {
+        "amenity": "restaurant",
+        "brand": "Rolls",
+        "brand:wikidata": "Q3195068",
+        "brand:wikipedia": "en:Rolls (restaurant chain)",
```
Comment (Identitaet) [inline data/brands/amenity/restaurant.json:2624]: If it's a finish brand we should use finish Wikipedia

**Review by Identitaet — CHANGES_REQUESTED** [review]

File: data/brands/amenity/restaurant.json (hunk 1 above)
Comment (KennethYoel) [reply data/brands/amenity/restaurant.json:2624]: I have thought the same as well but when I searched for Rolls in Finnish Wikipedia the page doesn't exist, only within the English Wikipedia that you can find that restaurant.

Comment (bhousel) [reply data/brands/amenity/restaurant.json:2624]: The [wikidata page for Rolls](https://www.wikidata.org/wiki/Q3195068) shows that there is one:
(this is where I usually go to look it up)

Screen Shot 2020-10-06 at 9 57 01 AM

Comment (KennethYoel) [reply data/brands/amenity/restaurant.json:2624]: 🤦🏻‍♂️ Thank you, I’ll get that updated.

**Comment by bhousel** [comment]
Thanks @KennethYoel and welcome to the project 👋

**Pull request merged**